packages/components/src/components/primitives/Snackbar/Snackbar.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect } from 'react';
+import { ToastContainer, toast } from 'react-toastify';
+import { getStyle } from './Snackbar.styles';
+import {
+  Typography,
+  TypographyMode,
+  TypographyTypes,
+} from 'components/primitives/Typography';
+import { Icon } from 'components/primitives/Icon';
+
+export const SNACKBAR_DEFAULT_TEST_ID = 'snackbar';
+
+export enum SnackbarType {
+  Default = 'default',
+  Success = 'success',
+  Warning = 'warning',
+  Error = 'error',
+}
+
+export enum SnackbarTextPosition {
+  Left = 'left',
+  Right = 'right',
+}
+
+export enum SnackbarPosition {
+  TopCenter = 'top-center',
+  TopRight = 'top-right',
+  TopLeft = 'top-left',
+  BottomRight = 'bottom-right',
+  BottomCenter = 'bottom-center',
+  BottomLeft = 'bottom-left',
+}
+
+type Props = {
+  testId?: string;
+  open?: boolean;
+  message: string;
+  description?: string;
+  showIcon?: boolean;
+  icon?: React.ReactNode;
+  textPosition?: SnackbarTextPosition;
+  showButton?: boolean;
+  buttonText?: string;
```

Comment:
what if we wanna show an icon instead?

Reply:
If we want to show an icon we have the `showIcon` and `icon` properties. We could create different versions for those.